code/edgebridge/src/main/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtension.java

```diff
@@ -81,6 +56,17 @@ protected String getVersion() {
 		return EdgeBridgeConstants.EXTENSION_VERSION;
```

Comment:
Has `EdgeBridgeConstants.EXTENSION_VERSION` been changed to "2.0.0" in a different PR? Or do you plan to do it as a separate change? 
Same question about `rootProject.mavenCoreVersion`